concent_api/utils/tests/test_decorators.py

```diff
@@ -229,3 +230,19 @@ def dummy_view_handle_http_response_not_allowed(_request, _message, _client_publ
 
         self.assertEqual(response.status_code, 405)
         self.assertEqual(len(response.content), 0)
+
+    @override_settings(
+        DEBUG=True
+    )
```

Comment:
A test is not effective if you're checking only one condition. You should check both the positive and the negative case in unit tests.

Reply:
Right, however after change above it will not be an issue anymore.